distributed/cli/dask_spec_worker.py

```diff
@@ -0,0 +1,37 @@
+import asyncio
+import click
+import json
+import yaml
+
+from distributed.deploy.spec import run_workers
+
+
+@click.command(context_settings=dict(ignore_unknown_options=True))
+@click.argument("scheduler", type=str, required=False)
+@click.option("--text", type=str, default="", help="")
```

Comment:
perhaps `--spec` and `--spec-file`? `--text` feels like the wrong name to me.

Reply:
I agree, something like `--spec` sounds more appropriate. Maybe `--spec` here and `--spec-file` (instead of `--file` in the line just below)?

Reply:
Done